builder/vsphere/driver/vm.go

```diff
@@ -580,6 +581,30 @@ func (vm *VirtualMachineDriver) Configure(config *HardwareConfig) error {
 		}
 		confSpec.DeviceChange = append(confSpec.DeviceChange, spec)
 	}
+
+	if config.Displays != 0 {
```

Comment:
You mentioned that if the value of Displays is greater than 10 the API will report an invalid configuration. Is the failure immediate as in the failure will happen before creating a vm?   

Or will the error occur at some point in time during a build? 

I'm trying to understand if we should have a validation for display in [here](https://github.com/hashicorp/packer-plugin-vsphere/blob/9500011375325c6f9633e42ba25090b7584a9880/builder/vsphere/common/step_hardware.go#L50) to fail early.

Reply:
Adding he validation would be nice however. In fact, technically it should be there for VideoRAM because there is a max of 256MB in vSphere 7.

Reply:
> It would fail immediately during the creation of the build if > 10. I considered adding a validation but the number could change between vSphere versions, thus I left it to the API response - which isn't pretty. :)

Good call! For some of the other builders some of these failures don't occur until late in the build. So folks would end up waiting a bit before it eventually fails. This is good to know for future things as well. Thanks for the context.

Reply:
Since the maximums are managed by the API based on the version of the virtual hardware, I've added a link to the product documentation and included the defaults.

```go
// Amount of video memory in KB. [See vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-789C3913-1053-4850-A0F0-E29C3D32B6DA.html)
// for supported maximums. Defaults to 4096 KB. 
VideoRAM int64 `mapstructure:"video_ram"`
// Number of video displays. [See vSphere documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-789C3913-1053-4850-A0F0-E29C3D32B6DA.html)
// for supported maximums. Defaults to 1. 
Displays int32 `mapstructure:"displays"`
```